showcase/src/app/welcome/pages/home/_home-page-theme.scss

```diff
@@ -0,0 +1,133 @@
+.centered-title {
+  text-align: center;
+  font-size: 17px;
+  line-height: 30px;
+  font-weight: 500;
+}
+
+.homepage-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 16px auto;
+}
+
+.doc-subtitle {
+  margin-left: 25px;
+}
+
+.stark-logo{
+  background-image: url(/assets/stark-core/logo/dark/stark_logo_dark.svg);
+  width: 250px;
+  height: 100px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  display: flex;
+  margin: auto;
+}
+
+.mat-icon.homepage-icons{
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
+}
+
+.mat-icon.core-image {
+  color: mat-color($accent-palette, 500);
+}
+
+.mat-icon.about-image {
+  color: mat-color($primary-palette, 500);
+}
+
+.mat-icon.ui-image {
+  color: mat-color($success-palette, 500);
+}
+
+.mat-icon.showcase-image {
+  color: mat-color($warning-palette, 900);
+}
+
+.page-content {
+  margin: 0;
+}
+
+.doc-description {
+  display: flex;
+  margin-left: 25px;
+  margin-right: 25px;
+
+  p {
+	display: flex;
+  }
+}
+
+.custom-button {
```

Comment:
Can you refactor the styles for `.custom-buttom`, `.custom-buttons` and `.buttons-row` so that it reflects the structure that they have in the HTML? This way I think you can clean up some unnecessary code and make it readable ;)